src/app/tests/suites/commands/interaction_model/InteractionModel.h

```diff
@@ -249,10 +249,12 @@ class InteractionModelCommands
         while (repeat--)
         {
 
-            chip::app::CommandPathParams commandPath = { endpointId, clusterId, commandId,
-                                                         (chip::app::CommandPathFlags::kEndpointIdValid) };
-            auto commandSender                       = std::make_unique<chip::app::CommandSender>(
-                mCallback, device->GetExchangeManager(), mTimedInteractionTimeoutMs.HasValue(), mSuppressResponse.ValueOr(false));
+            chip::app::CommandPathParams commandPath                = { endpointId, clusterId, commandId,
+                                                                        (chip::app::CommandPathFlags::kEndpointIdValid) };
+            std::unique_ptr<chip::app::CommandSender> commandSender = std::make_unique<chip::app::CommandSender>(
```

Comment:
Why is this not using `auto`?  This is the canonical case for `auto`: the entire type is already spelled out on the right-hand side of the assignment.

Reply:
Yes, this can be an auto. It was under an if else block before with the declaration before that, but now after bringing the AllowLargePayload() call inline it is back to a one-liner and should be an auto. Will change it.